app/src/main/java/com/bunbeauty/papakarlo/feature/auth/screen/login/LoginViewModel.kt

```diff
@@ -115,25 +128,41 @@ class LoginViewModel(
     }
 
     fun onNextClick() {
```

Comment:
I think it's better to add some event about code sending

Reply:
+